Kentico/kentico-cloud-docs-web
# KA-515 Introduce stages to Travis configuration

### Motivation

The goal of this change is further improve Travis build and deployment process. (KA-515)

### Checklist


### How to test

Please check the following:
- "predeployment tasks" are run in parallel
- deployment is not even initialized on branches other than develop or master
- ESLint fail -> deployment cancelled
